slack_sdk/web/internal_utils.py

```diff
@@ -229,7 +229,7 @@ def _next_cursor_is_present(data) -> bool:
 
 def _to_0_or_1_if_bool(v: Any) -> Union[Any, str]:
     if isinstance(v, bool):
-        return "1" if v else "0"
+        return 1 if v else 0
```

Comment:
Probably, this change is fine for WebClient etc. but I will check the behavior later.

Reply:
As `WebookClient` sends `content-type: application/json` data, we don't necessarily convert boolean values to 1/0. Can you modify `slack_sdk/webhook/internal_utils.py` this way instead?

```diff
diff --git a/slack_sdk/webhook/internal_utils.py b/slack_sdk/webhook/internal_utils.py
index 37def28..091c06c 100644
--- a/slack_sdk/webhook/internal_utils.py
+++ b/slack_sdk/webhook/internal_utils.py
@@ -12,7 +12,6 @@ from .webhook_response import WebhookResponse
 def _build_body(original_body: Optional[Dict[str, Any]]) -> Optional[Dict[str, Any]]:
     if original_body:
         body = {k: v for k, v in original_body.items() if v is not None}
-        body = convert_bool_to_0_or_1(body)
         _parse_web_class_objects(body)
         return body
     return None
```

Reply:
@seratch. Okay, I see that changing it this way reduces the number of possible affected surfaces. Thanks! I have also reverted the above line to its original form and made the suggested change.